src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -830,4 +837,34 @@ def log(self, message: LoggableType | list[LoggableType], name: str | None = Non
 
     def _append_tool_output(self):
         if next((output for output in self.outputs if output.name == TOOL_OUTPUT_NAME), None) is None:
-            self.outputs.append(Output(name=TOOL_OUTPUT_NAME, display_name="Tool", method="to_toolkit", types=["Tool"]))
+            set_as_tool_input = self._inputs.get("set_as_tool")
+            hidden_value = not set_as_tool_input.value if set_as_tool_input is not None else False
```

Comment:
Can we simplify this to
```
hidden_value = set_as_tool_input and not set_as_tool_input.value
```

Reply:
I dont think so. But let me try an alternate approach